### DISCUSSION TITLE
How do I verify the identity of users on the backend?

### DISCUSSION THREAD

**Discussion by DeepBhat** [appwrite/appwrite#7286] [Q&A] [answered]
Hey there,

I have some custom endpoints and I want to implement authorization on those endpoints. What is the best approach to verify the identity of  the user on the  backend  (and get the user ID too)? Using  JWT works for  me,  but I  was  wondering if we can also use secure cookies for the same

**Comment by stnguyen90** [comment, marked as answer]
On the backend, you'd have to make the Get Account API call, but you need a JWT or the session cookie. You can find more info about these in the [REST API docs](https://appwrite.io/docs/apis/rest).
